apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.scss

```diff
@@ -32,4 +32,35 @@
     right: 1.5rem;
     top: calc(50% - 10px);
   }
+
+  .custom-button {
```

Comment:
`.drop-area {`

Reply:
Done. Should I change `.button-content` to `.drop-area-content` then as well?

Reply:
@dtslvr Looks like you missed this one :)